icons/euro.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  xmlns="http://www.w3.org/2000/svg"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M16 10.5H6" />
+  <path d="M15 13.5H6" />
+  <path d="M18 17.545c-.898.675-1.993.955-3.118.955-3.235 0-6.382-3-6.382-6.5s3.147-6.5 6.382-6.5c1.125 0 2.22.25 3.118.93" />
```

Comment:
```suggestion
  <path d="M4 10h12" />
  <path d="M4 14h9" />
  <path d="M19 6a7.7 7.7 0 00-5.2-2A7.9 7.9 0 006 12c0 4.4 3.5 8 7.8 8 2 0 3.8-.8 5.2-2" />
```

I refactored the euro sign a bit. So it matches more with the rest. (top left is the original)
![image](https://user-images.githubusercontent.com/11825403/109313599-acfa5f00-7848-11eb-9ccb-100863b06c45.png)